src/aws-exports.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
+// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
```

Comment:
Is this actually an auto-generated file, or copy-pasted into the project from an auto-generated original?

Reply:
It's copy-pasted from the autogenerated original, yes. I'll defer to your judgement about whether it makes sense for the comment to exist or not.

Reply:
It's fine, just curious. 😉